v1.3.2 and latest commit from master branch no longer build on Arch Linux

Hello, I'm the maintainer of the AUR pkgbuilds popsicle and popsicle-git

**Distribution (run `cat /etc/os-release`):**
 
```
$ cat /etc/os-release
NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://bugs.archlinux.org/"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo
```
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

v1.3.2 and latest commit from master branch

**Issue/Bug Description:**

Both the last stable release and last commit no longer build on Arch Linux, this has been reported to me today and have confirmed the issue

Rust is at v1.74.1


v1.3.2
```
   Compiling futures_codec v0.4.1
error: failed to run custom build command for `loopdev v0.4.0`

Caused by:
  process didn't exit successfully: `/home/fabio/Dev/Github/PKGBUILD-AUR_fix/p/popsicle/src/popsicle/target/release/build/loopdev-13cf59a56d32320a/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at /home/fabio/Dev/Github/PKGBUILD-AUR_fix/p/popsicle/src/popsicle/vendor/proc-macro2/src/fallback.rs:791:9:
  "enum_(unnamed_at_/usr/include/linux/loop_h_16_1)" is not a valid Ident
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:107: target/release/popsicle-gtk] Error 101
==> ERROR: A failure occurred in build().
```

v1.3.2.r4.g22c34ae
```
   Compiling num_enum v0.5.11
error: failed to run custom build command for `loopdev v0.4.0`

Caused by:
  process didn't exit successfully: `/home/fabio/Dev/Github/PKGBUILD-AUR_fix/p/popsicle-git/src/popsicle/target/release/build/loopdev-13cf59a56d32320a/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at /home/fabio/Dev/Github/PKGBUILD-AUR_fix/p/popsicle-git/src/popsicle/vendor/proc-macro2/src/fallback.rs:791:9:
  "enum_(unnamed_at_/usr/include/linux/loop_h_16_1)" is not a valid Ident
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:107: target/release/popsicle-gtk] Error 101
==> ERROR: A failure occurred in build().
```


**Steps to reproduce (if you know):**

Install Arch Linux
sudo pacman -Syyu base-devel git
git clone https://aur.archlinux.org/popsicle.git
cd popsicle
makepkg -s

popsicle-git on AUR still have old build instructions at the moment, realized now when checking, I have updated it locally for now

**Expected behavior:**

pkgbuilds build fine

**Other Notes:**

Still doesn't build:

```
   Compiling loopdev v0.4.0
error: failed to run custom build command for `loopdev v0.4.0`

Caused by:
  process didn't exit successfully: `/build/popsicle/src/popsicle/target/release/build/loopdev-2def6cd8e1102345/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at /build/popsicle/src/popsicle/vendor/bindgen/src/ir/context.rs:878:9:
  "enum_(unnamed_at_/usr/include/linux/loop_h_16_1)" is not a valid Ident
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:107: target/release/popsicle-gtk] Error 101

```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -19,17 +19,18 @@ path = "src/lib.rs"
 members = ["cli", "gtk"]
 
 [dependencies]
-anyhow = "1.0.65"
+anyhow = "1.0.79"
 as-result = "0.2.1"
 async-std = "1.12.0"
 derive-new = "0.6.0"
-futures = "0.3.24"
+futures = "0.3.30"
 futures_codec = "0.4.1"
-libc = "0.2.134"
-memchr = "2.5.0"
+libc = "0.2.151"
+memchr = "2.7.1"
 mnt = "0.3.1"
-ron = "0.8.0"
-serde = { version = "1.0.145", features = ["derive"] }
+ron = "0.8.1"
+serde = { version = "1.0.194", features = ["derive"] }
 srmw = "0.1.1"
-thiserror = "1.0.37"
+thiserror = "1.0.56"
 usb-disk-probe = "0.2.0"
+

--- a/cli/Cargo.toml
+++ b/cli/Cargo.toml
@@ -12,19 +12,19 @@ name = "popsicle"
 path = "src/main.rs"
 
 [dependencies]
-anyhow = "1.0.65"
+anyhow = "1.0.79"
 async-std = "1.12.0"
 atty = "0.2.14"
 better-panic = "0.3.0"
 cascade = "1.0.1"
-clap = "4.0.8"
+clap = "4.4.13"
 derive-new = "0.6.0"
-fomat-macros = "0.3.1"
-futures = "0.3.24"
+fomat-macros = "0.3.2"
+futures = "0.3.30"
 i18n-embed = { version = "0.14.1", features = ["fluent-system", "desktop-requester"] }
 i18n-embed-fl = "0.7.0"
-libc = "0.2.134"
-once_cell = "1.15.0"
+libc = "0.2.151"
+once_cell = "1.19.0"
 # pbr = "1.0.4"
 pbr = { git = "https://github.com/ids1024/pb", branch = "write" }
 popsicle = { path = ".." }

--- a/gtk/Cargo.toml
+++ b/gtk/Cargo.toml
@@ -13,32 +13,32 @@ path = "src/main.rs"
 
 [dependencies]
 atomic = "0.6.0"
-anyhow = "1.0.65"
+anyhow = "1.0.79"
 bytemuck = "1.14.0"
-bytesize = "1.1.0"
+bytesize = "1.3.0"
 cascade = "1.0.1"
-crossbeam-channel = "0.5.6"
-dbus = "0.9.6"
+crossbeam-channel = "0.5.10"
+dbus = "0.9.7"
 dbus-udisks2 = { git = "https://github.com/pop-os/dbus-udisks2" }
-digest = "0.10.5"
-futures = "0.3.24"
-gdk = "0.17.0"
-gdk-pixbuf = "0.17.0"
-gio = "0.17.0"
-glib = "0.17.0"
-gtk = { version = "0.17.0" }
+digest = "0.10.7"
+futures = "0.3.30"
+gdk = "0.17.1"
+gdk-pixbuf = "0.17.10"
+gio = "0.17.10"
+glib = "0.17.10"
+gtk = { version = "0.17.1" }
 hex-view = "0.1.3"
 iso9660 = { git = "https://github.com/ids1024/iso9660-rs" }
-libc = "0.2.134"
-md-5 = "0.10.5"
-pango = "0.17.0"
+libc = "0.2.151"
+md-5 = "0.10.6"
+pango = "0.17.10"
 popsicle = { path = ".." }
 pwd = "1.4.0"
-sha2 = "0.10.6"
-sha-1 = { version = "0.10.0", features = ["asm"] }
-sys-mount = "2.0.2"
+sha2 = "0.10.8"
+sha-1 = { version = "0.10.1", features = ["asm"] }
+sys-mount = "2.1.1"
 async-std = "1.12.0"
 i18n-embed = { version = "0.14.1", features = ["fluent-system", "desktop-requester"] }
 i18n-embed-fl = "0.7.0"
 rust-embed = { version = "8.2.0", features = ["debug-embed"] }
-once_cell = "1.15.0"
+once_cell = "1.19.0"

